contracts/Aavegotchi/facets/PeripheryFacet.sol

```diff
@@ -0,0 +1,90 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.1;
+
+import "../libraries/LibItems.sol";
+import "../libraries/LibAppStorage.sol";
+import "../libraries/LibERC1155Marketplace.sol";
+import "../../shared/libraries/LibStrings.sol";
+
+contract PeripheryFacet is Modifiers {
+    ///READ FUNCTIONS
+
+    function peripheryBalanceOf(address _owner, uint256 _id) external view returns (uint256 balance_) {
```

Comment:
I don't think a special periphery function is needed for these view functions that don't rely on msg sender.

Reply:
yea....makes sense